src/Components/App/App.test.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { render, fireEvent, waitFor } from '@testing-library/react';
+import { render, fireEvent, waitFor, act } from '@testing-library/react';
```

Comment:
the docs recommend importing "act" from the react library and not test-utils. This may help the current errors. Also, we can remove react-dom/test-utils from dependencies.